pandas/tests/series/methods/test_convert_dtypes.py

```diff
@@ -226,3 +226,12 @@ def test_convert_bool_dtype(self):
         # GH32287
         df = pd.DataFrame({"A": pd.array([True])})
         tm.assert_frame_equal(df, df.convert_dtypes())
+
+    def test_convert_byte_string_dtype(self):
+        # GH43183
+        byte_str = b"binary-string"
+
+        df = pd.DataFrame(data={"A": byte_str}, index=[0])
+        df_convert = df.convert_dtypes()
+
+        assert df_convert["A"][0].decode("ascii") == "binary-string"
```

Comment:
can you construct the expected frame and use `tm.assert_frame_equal(result, expected)`

Reply:
Done